plugin/src/main/java/org/opennms/velocloud/requisition/PartnerRequisitionProvider.java

```diff
@@ -62,6 +70,63 @@ protected Requisition handleRequest(final Request request, final VelocloudApiCli
+                            .setForeignId(gateway.gatewayId)
+                            .setNodeLabel(gateway.gatewayName)
+                            .setLocation(gateway.siteName)
+                            .setInterfaces(List.of(ImmutableRequisitionInterface.newBuilder()
+                                    .setIpAddress(gateway.ipAddress)
+                                    .build()))
+                            .addMetaData(ImmutableRequisitionMetaData.newBuilder()
+                                    .setContext(VELOCLOUD_CONTEXT)
+                                    .setKey("roles")
+                                    .setValue(users.stream().map(u ->
+                                            String.format("%s:%s", u.roleId, u.roleName)).collect(Collectors.joining(",")))
+                                    .build())
+                            .addMetaData(ImmutableRequisitionMetaData.newBuilder()
+                                    .setContext(VELOCLOUD_CONTEXT)
+                                    .setKey("deviceId")
+                                    .setValue(gateway.deviceId)
+                                    .build())
+                            .addMetaData(ImmutableRequisitionMetaData.newBuilder()
+                                    .setContext(VELOCLOUD_CONTEXT)
+                                    .setKey("gatewayId")
+                                    .setValue(gateway.gatewayId)
+                                    .build())
+                            .addMetaData(ImmutableRequisitionMetaData.newBuilder()
+                                    .setContext(VELOCLOUD_CONTEXT)
+                                    .setKey("gatewayName")
+                                    .setValue(gateway.gatewayName)
+                                    .build())
+                            .addMetaData(ImmutableRequisitionMetaData.newBuilder()
+                                    .setContext(VELOCLOUD_CONTEXT)
+                                    .setKey("enterpriseId")
+                                    .setValue(gateway.enterpriseId.toString())
+                                    .build())
+                            .addMetaData(ImmutableRequisitionMetaData.newBuilder()
+                                    .setContext(VELOCLOUD_CONTEXT)
+                                    .setKey("buildNumber")
+                                    .setValue(gateway.buildNumber)
+                                    .build())
+                            .addMetaData(ImmutableRequisitionMetaData.newBuilder()
+                                    .setContext(VELOCLOUD_CONTEXT)
+                                    .setKey("softwareVersion")
+                                    .setValue(gateway.softwareVersion)
+                                    .build())
+                            .build());
+                });
+
+            }
+        } catch (VelocloudApiException e) {
+            LOG.error(e.getMessage());
```

Comment:
```suggestion
            LOG.error("Building requisition failed" ,e);
```

By adding the exception without a placeholder in the message (must be last parameter), we get the whole stacktrace in the logs.